pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -395,8 +401,23 @@ func (adc *attachDetachController) GetDesiredStateOfWorld() cache.DesiredStateOf
 }
 
 func (adc *attachDetachController) podUpdate(oldObj, newObj interface{}) {
```

Comment:
Also, regardless of this event though - I have also updated DSWP to remove the pod from DSW if pod is terminated.